pkg/dhclient/dhclient.go

```diff
@@ -140,7 +140,9 @@ func lease4(ctx context.Context, iface netlink.Link, timeout time.Duration, retr
 	}
 
 	log.Printf("Attempting to get DHCPv4 lease on %s", iface.Attrs().Name)
-	_, p, err := client.Request(ctx, dhcpv4.WithNetboot)
+	_, p, err := client.Request(ctx, dhcpv4.WithNetboot,
+		dhcpv4.WithOption(dhcpv4.OptGeneric(dhcpv4.OptionClassIdentifier, []byte("PXE UROOT"))),
```

Comment:
Use https://godoc.org/github.com/insomniacslk/dhcp/dhcpv4#OptClassIdentifier instead of generic

Reply:
Done